modules/currency-l0/src/main/scala/org/tessellation/currency/ExampleCurrencyL0.scala

```diff
@@ -0,0 +1,16 @@
+package org.tessellation.currency
+
+import java.util.UUID
+
+import org.tessellation.currency.schema.currency.TokenSymbol
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.cluster.ClusterId
+
+import eu.timepit.refined.auto._
+
+object ExampleCurrencyL0
```

Comment:
do we need this in the currency module? shouldn't it be some kind of example project?

Reply:
For now I decided to keep it there and wait for @TheMMaciek's changes. Then we will make proper modules and structure the project.